qiskit/tools/jupyter/__init__.py

```diff
@@ -106,7 +98,6 @@
 from .progressbar import HTMLProgressBar
 from .version_table import VersionTable
 from .copyright import Copyright
-from .monospace import MonospacedOutput
```

Comment:
This should be deprecated before being removed. We also just added this last release it's a pretty bad user experience to say "hey, here's a new thing you can use to adjust the text drawer output" Then 2-3 months later "that new thing is going away"